pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -287,6 +287,7 @@ func TestDesiredRouterDeployment(t *testing.T) {
 	checkDeploymentHasEnvVar(t, deployment, "ROUTER_CAPTURE_HTTP_REQUEST_HEADERS", false, "")
 	checkDeploymentHasEnvVar(t, deployment, "ROUTER_CAPTURE_HTTP_RESPONSE_HEADERS", false, "")
 	checkDeploymentHasEnvVar(t, deployment, "ROUTER_CAPTURE_HTTP_COOKIE", false, "")
+	checkDeploymentHasEnvVar(t, deployment, "ROUTER_DONT_LOG_NULL", false, "")
```

Comment:
Can we reference the constant (`RouterDontLogNull`) for `"ROUTER_DONT_LOG_NULL"`? And ditto for "ROUTER_HTTP_IGNORE_PROBES".

Reply:
Done.